visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -102,22 +102,15 @@ export class CodeMapRenderService implements IsLoadingFileSubscriber {
 
 		for (const node of sortedNodes) {
 			if (node.isLeaf) {
-				switch (node.color) {
-					case mapColor.negative:
-						this.nodesByColor.negative.push(node)
-						break
-
-					case mapColor.positive:
+			    const metric = node.attributes[dynamicSettings.colorMetric]
+			    if (metric != null) {
```

Comment:
```suggestion
			    if (metric == null) continue
```
This should do the same, but we can avoid nesting ifs and keep our own code metrics looking good.